platform/security/handler/security-handler-anonymous/src/main/java/org/codice/ddf/security/handler/anonymous/AnonymousHandler.java

```diff
@@ -71,7 +71,7 @@ public HandlerResult getNormalizedToken(ServletRequest request, ServletResponse
         HandlerResult result = new HandlerResult();
```

Comment:
The auth type should be changed from ANON to guest.  Making it clear to admins that this authentication type is for guests was one of the main goals for renaming.